octobot_trading/exchanges/connectors/ccxt_exchange.py

```diff
@@ -171,12 +163,24 @@ def _unauthenticated_exchange_fallback(self, err):
         self.client = self._get_unauthenticated_exchange()
 
     def _get_unauthenticated_exchange(self):
-        return self.exchange_type({
+        return self.exchange_type(self._get_client_config())
+
+    def _get_client_config(self, api_key=None, secret=None, password=None):
```

Comment:
Can you add a cython declaration like `_get_unauthenticated_exchange` please ?

Reply:
sure :D